index.html

```diff
@@ -10,125 +10,143 @@
   <body>
     <header>
       <a id="logo">Farida's Logo</a>
+      <nav id="top">
+        <a>Portfolio</a>
+        <a>About</a>
+        <a>Contact</a>
+      </nav> 
       <nav>
         <img src="./images/Union.jpg" alt="menue bar" id="menue-button">
       </nav>
     </header>
     <main>
       <section id="intro">
-        <h1>I'm Farida Glad to see you!</h1>
-        <p id="bio">I’m a software developer! I can help you build a product, 
-          feature or website Look through some of my work and experience! 
-          If you like what you see and have a project you need coded, 
-          don’t hestiate to contact me.
-        </p>
-        
-        <h3>let's connect</h3>
-        <ul class="social">
-          <li><img src="./images/Vector.png" alt="twitter"></li>
-          <li><img src="./images/Linkedin-icon.jpg" alt="linkedin"></li>
-          <li><img src="./images/Vector.jpg" alt="email"></li>
-          <li><img src="./images/Vector-2.jpg" alt="github"></li>
-          <li><img src="./images/Vector-3.jpg" alt="angel-list"></li>
-        </ul>
+        <div id="intro-content">
+          <h1>I'm Farida</h1>
+          <h1>Glad to see you!</h1>
+          <p>I’m a software developer! I can help you build a product, 
+            feature or website Look through some of my work and experience! 
+            If you like what you see and have a project you need coded, 
+            don’t hestiate to contact me.
+          </p>
+          
+          <h3>let's connect</h3>
+          <ul class="social">
+            <li><img src="./images/Vector.png" alt="twitter"></li>
+            <li><img src="./images/Linkedin-icon.jpg" alt="linkedin"></li>
+            <li><img src="./images/Vector.jpg" alt="email"></li>
+            <li><img src="./images/Vector-2.jpg" alt="github"></li>
+            <li><img src="./images/Vector-3.jpg" alt="angel-list"></li>
+          </ul>
```

Comment:
- [ ] [Optional] Social media icons must be wrapped in a `a` tag, because they are _hyperlink_ to each one of your social profiles. I strongly suggest you wrap each element in a `a` tag